app.yaml

```diff
@@ -63,6 +63,11 @@ handlers:
   http_headers:
     Cache-Control: 'public, max-age=2592000'
     Vary: Accept-Encoding
+- url: /i18n
+  static_dir: i18n/
```

Comment:
Following existing conventions in this file, let's omit the slash after i18n for the 'static_dir' field.